airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -163,6 +164,8 @@ def __init__(self, context: ConnectorContext) -> None:
             context=context,
             paths_to_mount=[
                 MountPath(code_directory),
+                # Support airbyte-enterprise docs.
+                MountPath(SUBMODULE_ENTERPRISE_DOCS_PATH),
```

Comment:
The non existence of this path in the OSS repo makes leads to workflow failure:
```
Traceback (most recent call last):
  File "/opt/pipx/venvs/pipelines/lib/python3.10/site-packages/pipelines/airbyte_ci/connectors/test/commands.py", line 172, in test
    await run_connectors_pipelines(
  File "/opt/pipx/venvs/pipelines/lib/python3.10/site-packages/pipelines/airbyte_ci/connectors/pipeline.py", line 109, in run_connectors_pipelines
    async with anyio.create_task_group() as tg_connectors:
  File "/opt/pipx/venvs/pipelines/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 597, in __aexit__
    raise exceptions[0]
  File "/opt/pipx/venvs/pipelines/lib/python3.10/site-packages/pipelines/airbyte_ci/connectors/test/pipeline.py", line 61, in run_connector_test_pipeline
    StepToRun(id=CONNECTOR_TEST_STEP_ID.QA_CHECKS, step=QaChecks(context)),
  File "/opt/pipx/venvs/pipelines/lib/python3.10/site-packages/pipelines/airbyte_ci/connectors/test/steps/common.py", line 168, in __init__
    MountPath(SUBMODULE_ENTERPRISE_DOCS_PATH),
  File "<string>", line 5, in __init__
  File "/opt/pipx/venvs/pipelines/lib/python3.10/site-packages/pipelines/models/steps.py", line 60, in __post_init__
    self._check_exists()
  File "/opt/pipx/venvs/pipelines/lib/python3.10/site-packages/pipelines/models/steps.py", line 53, in _check_exists
    raise FileNotFoundError(message)
FileNotFoundError: airbyte-submodule/docs/integrations/enterprise-connectors does not exist.
```

I think this is because `MountPath` has checks the existence of the path in its `__post_init__`. I would suggest dynamically building the `paths_to_mount` list to conditionally add the submodule docs path if it exists...